src/util/mutations.js

```diff
@@ -56,9 +58,8 @@ export const onNewPosts = Object.freeze({
 const onBeforeRepaint = () => {
   repaintQueued = false;
 
-  const addedNodes = mutationsPool
+  const addedNodes = addedNodesPool
     .splice(0)
-    .flatMap(({ addedNodes }) => [...addedNodes])
     .filter(addedNode => addedNode instanceof Element && addedNode.isConnected);
```

Comment:
We don't need to repeat `addedNode instanceof Element` here, that filtering is already performed in the observer callback
```suggestion
    .filter(addedNode => addedNode.isConnected);
```

Reply:
Oops—thought I had removed that!

Reply:
(I considered changing `addedNode` to `addedElement`, which we could also do.